src/components/PathwayProvider.tsx

```diff
@@ -27,34 +27,44 @@ interface PathwayProviderProps {
   children: ReactNode;
 }
 
-export const PathwayProvider: FC<PathwayProviderProps> = memo(({ children }) => {
-  const [pathways, setPathways] = useState<Pathway[]>([]);
+export const PathwayProvider: FC<PathwayProviderProps> = memo(function PathwayProvider({
```

Comment:
Now that there's also a `CurrentPathwayProvider` the meaning of the name `PathwayProvider` becomes less clear. Can this be renamed to the plural Pathway**s**Provider to make the distinction more obvious? I'm also ok with making that a separate task/PR to reduce the number of changes here.

Reply:
Let me look into it after to burndown - hopefully its just a matter of changing all the names which is easy

Reply:
Renamed easily